wrappers/swift/LogConfiguration.swift

```diff
@@ -0,0 +1,214 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Class wrapping `ODWLogConfiguration` ObjC class object, representing configuration related to events.
+public class LogConfiguration {
```

Comment:
Do we expect clients to subclass this? If not, we should mark this as `final` to prevent unexpected behaviors.